tests/test_job.py

```diff
@@ -500,3 +500,24 @@ def test_job_resubmit_all(hq_env: HqEnv):
 
     table = hq_env.command(["resubmit", "1"], as_table=True)
     assert table[3] == ["Tasks", "3; Ids: 2, 7, 9"]
+
+
+def test_job_wait(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.start_worker()
+    hq_env.command(["submit", "sleep", "1"])
+    r = hq_env.command(["wait", "1"])
+    assert "Waiting for 1 job(s)" in r
+    table = hq_env.command(["job", "1"], as_table=True)
+    assert table[2][1] == "FINISHED"
```

Comment:
There is a new API for reading the table outputs. Using numeric indices is annoying, since after each table change we have to change a lot of tests. Please use this: https://github.com/It4innovations/hyperqueue/blob/00d595f3659ae7ec326f214c8fc6797704c4dab8/tests/utils.py#L31 :innocent: